CHANGELOG.md

```diff
@@ -35,6 +35,8 @@
   Add `under2` and `underF2` functions to `Relude.Extra.Newtype`.
 * [#60](https://github.com/kowainik/relude/issues/60):
   Add `hoistMaybe` and `hoistEither` functions.
+* [#81](https://github.com/kowainik/relude/issues/81):
```

Comment:
Thanks a lot for not forgetting to update the CHANGELOG! Could you please create separate entry with `0.3.1` version and move this line there? And also update version in `.cabal` file? `0.3.0` version is already published to Hackage, so this change is going to the next version.

Reply:
Ah right